client/dist/style.css

```diff
@@ -2,6 +2,11 @@
   max-width: 500px;
   max-height: 500px;
 }
+.styles {
```

Comment:
I started prepending my classNames/id's with "rr-" to indicate my component (Ratings and Reviews), I think we should all do that in case we have similar naming.  You'll see it in my next PR.